src/app/shared/services/helper.service.ts

```diff
@@ -176,8 +180,8 @@ export class HelperService {
     parent.items.push(node);
   }
 
-  getSelectedIds(reports: any[]): string[] {
-    let copiedIds: string[] = [];
+  getSelectedIds(reports: any[]): number[] {
+    let copiedIds: number[] = [];
     this.getSelectedReports(reports).forEach((report: Report) => copiedIds.push(report.storageId));
```

Comment:
While you're at it, can you refactor this to use a a `for...of` loop instead of a `foreach` loop?

Reply:
Added the change.